src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -385,6 +385,14 @@ enum AppStrings {
 		static let legalDetailsButtonTitle = NSLocalizedString("SurveyConsent_Legal_Details_Button_Title", comment: "")
 		static let legalDetailsTitle = NSLocalizedString("SurveyConsent_Legal_Details_Title", comment: "")
 		static let legalDetailsBody = NSLocalizedString("SurveyConsent_Legal_Details_Body", comment: "")
+
+		// Errors
+		static let errorTitle = NSLocalizedString("SurveyConsent_Error_Title", comment: "")
```

Comment:
String file got changed in an other PR?

Reply:
We have the strings on a separated branch.